Tests/Marketplace/zip_packs.py

```diff
@@ -114,61 +114,109 @@ def remove_test_playbooks_from_signatures(path, filenames):
-                executor_submit(executor, download_path, blob)
-                sleep(1)
-                if os.path.exists('/home/runner/work/content-private/content-private/content/artifacts/'):
-                    logging.info(f"Copying pack from {download_path} to /home/runner/work/content-private/"
-                                 f"content-private/content/artifacts/packs/{pack.name}.zip")
-                    shutil.copy(download_path,
-                                f'/home/runner/work/content-private/content-private/content/artifacts/'
-                                f'packs/{pack.name}.zip')
-            else:
-                logging.warning(f'Did not find a pack to download with the prefix: {pack_prefix}')
+    if gcp_path == BUILD_GCP_PATH:
+        zip_path = os.path.join(zip_path, 'packs')  # directory of the packs
+    else:
+        zip_path = os.path.join(zip_path, circle_build)  # directory of the packs
+
+    dir_entries = os.listdir(zip_path)
+    packs_list = [pack.name for pack in os.scandir(PACKS_FULL_PATH)]  # list of all packs from repo
+    print(f"List of packs in repo is {', '.join(packs_list)}")
+
+    for entry in dir_entries:
+        entry_path = os.path.join(zip_path, entry)
+        if entry not in IGNORED_FILES and entry in packs_list and os.path.isdir(entry_path):
+            # This is a pack directory, should keep only most recent release zip
+            pack_files = get_pack_files(entry_path)
+            latest_zip = get_latest_pack_zip_from_blob(entry, pack_files)
+            if not latest_zip:
+                logging.warning(f'Failed to get the zip of the pack {entry} from GCP')
+                continue
+            print(f"Found latest zip of {entry}, which is {latest_zip}")
+            zipped_packs.append({Path(latest_zip).stem: latest_zip})
 
     if not zipped_packs:
         logging.critical('Did not find any pack to download from GCP.')
         sys.exit(1)
     return zipped_packs
 
 
+def download_packs_from_gcp(storage_bucket_name, gcp_path, destination_path, circle_build, branch_name):
+    """
+    Copies all content in the gcp path to destination_path
+    Args:
+        storage_bucket_name: The name of the GCP bucket to download from.
+        gcp_path: The path of the packs in the GCP bucket.
+        destination_path: The path to download the packs to.
+        branch_name: The branch name of the build.
+        circle_build: The number of the circle ci build.
+    """
+
+    if gcp_path == BUILD_GCP_PATH:
+        src_path = os.path.join('gs://', storage_bucket_name, gcp_path, branch_name, circle_build, 'content', 'packs')
+    else:
+        src_path = os.path.join('gs://', storage_bucket_name, gcp_path, branch_name, circle_build)
+
+    if not branch_name or not circle_build:
+        src_path = src_path.replace('/builds/content', '')
+
+    try:
+        process = subprocess.Popen(["Tests/scripts/cp_gcp_dir.sh", src_path, destination_path])
```

Comment:
Use check_output which is a higher level api: https://docs.python.org/3/library/subprocess.html#subprocess.check_output

Reply:
Fixed, thanks.